components/eamxx/src/physics/ml_correction/eamxx_ml_correction_process_interface.cpp

```diff
@@ -22,40 +28,90 @@ void MLCorrection::set_grids(
   // Nevertheless, for output reasons, we like to see 'kg/kg'.
   auto Q = kg / kg;
   Q.set_string("kg/kg");
-
+  constexpr int ps = SCREAM_SMALL_PACK_SIZE;
```

Comment:
I'm curious, why do you pick the small pack size rather than the default one?

Reply:
Frankly no reason, I only learned about pack size recently. This is also used in the unit test, should it be the default pack size?

Reply:
In practice, it probably does not matter, since we default SMALL_PACK_SIZE=PACK_SIZE, but it does look a bit confusing. I think at the beginning we had a small pack size, which was more suitable for stuff with lots of conditionals (like microphysics). But unless you have lots of if statements concerning the vertical direction, you should use the largest pack size.